[Bug fix] Fix search

kcn3388 opened the pull request.

kcn3388 committed:
fix search

kcn3388 commented:
你贴一下正则表达式的报错呗，我修一下（）

SonderXiaoming commented:
![image](https://user-images.githubusercontent.com/98363578/183538420-f005418e-91fe-411a-84d4-aa6a6932a928.png)

SonderXiaoming commented:
<img width="541" alt="image" src="https://user-images.githubusercontent.com/98363578/183538610-6dfca782-72bc-47bb-b51e-b134db1330a9.png">
<img width="629" alt="image" src="https://user-images.githubusercontent.com/98363578/183538637-83feb7cb-bd3a-4b62-9082-159cc8de2fcc.png">

SonderXiaoming commented:
没想到吧，我yobot路径实际上是这样

kcn3388 committed:
fix style when cilck for help

kcn3388 commented:
> ![image](https://user-images.githubusercontent.com/98363578/183538420-f005418e-91fe-411a-84d4-aa6a6932a928.png)

（别脱完，就这样）
明白了，同时存在两个readme的时候的问题，修一下

SonderXiaoming commented:
> > ![image](https://user-images.githubusercontent.com/98363578/183538420-f005418e-91fe-411a-84d4-aa6a6932a928.png)
> 
> （别脱完，就这样） 所以你这个readme是在module外部吗（）

准确来说，应该是module里面直接包裹了一个文件夹，代码在那个文件夹里

kcn3388 commented:
> 准确来说，应该是module里面直接包裹了一个文件夹，代码在那个文件夹里

你需要子文件夹里面的readme还是父文件夹的readme（）
父文件夹的话很简单，第一次检测到readme就return就行
子文件夹的话就改写一下，把子文件夹的路径加上去就行（）

SonderXiaoming commented:
> > 准确来说，应该是module里面直接包裹了一个文件夹，代码在那个文件夹里
> 
> 你需要子文件夹里面的readme还是父文件夹的readme（） 父文件夹的话很简单，第一次检测到readme就return就行 子文件夹的话就改写一下，把子文件夹的路径加上去就行（）

我觉得一般是父文件夹的readme优先度高，直接return吧，（同一文件夹下我希望userreadme优先度最高，之前用列表直接排好序了）

kcn3388 committed:
fix bug

now won't get FileNotFoundError when readme both in father folder and child folder

kcn3388 commented:
因为文件名顺序上是readme更前，所以既要保证优先父文件夹又要先userreadme就只能蠢办法了（）
已经推了commit了，你试试（）

SonderXiaoming commented:
> 因为文件名顺序上是readme更前，所以既要保证优先父文件夹又要先userreadme就只能蠢办法了（） 已经推了commit了，你试试（）

可以了，你css改好了我就通过了

kcn3388 commented:
> 可以了，你css改好了我就通过了

css没问题啊，你ctrl+f5刷新一下静态资源就行了（）
[demo](https://clash.kcn3388.com/xcwhelp)
